Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt

```diff
@@ -11,11 +11,11 @@ import androidx.lifecycle.lifecycleScope
 import androidx.navigation.fragment.findNavController
 import com.paypal.android.R
 import com.paypal.android.api.services.SDKSampleServerApi
-import com.paypal.android.checkoutweb.PayPalWebCheckoutClient
-import com.paypal.android.checkoutweb.PayPalWebCheckoutFundingSource
-import com.paypal.android.checkoutweb.PayPalWebCheckoutListener
-import com.paypal.android.checkoutweb.PayPalWebCheckoutRequest
-import com.paypal.android.checkoutweb.PayPalWebCheckoutResult
+import com.paypal.android.webpayments.PayPalWebCheckoutClient
```

Comment:
🧶 : Same here, should the package name be `com.paypal.android.paypalcheckoutweb` to match the artifact id?